### PR TITLE
Add `getAttributes` on `Graph`

### PR DESCRIPTION
**Pull request by jxnu-liguobin**
References #31142

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
At least one commit author (li.guobin@immomo.com) is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

**Comment by lightbend-cla-validator** [comment]
At least one commit author (li.guobin@immomo.com, li.guobin@immomo.com) is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

**Comment by lightbend-cla-validator** [comment]
At least one commit author (li.guobin@immomo.com) is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

**Comment by lightbend-cla-validator** [comment]
At least one commit author (li.guobin@immomo.com) is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

**Comment by johanandren** [comment]
Something weird with the CI server, I'll try closing and reopening again and see if it retriggers.

**Comment by johanandren** [comment]
Nope, that didn't help, not sure what is going on, @jxnu-liguobin can you push something to retrigger (squash or just some dummy commit would be fine)

**Comment by jxnu-liguobin** [comment]
> Nope, that didn't help, not sure what is going on, @jxnu-liguobin can you push something to retrigger (squash or just some dummy commit would be fine)

ok

**Comment by hepin1989** [comment]
@johanandren seems like a maintainer to approve for the first contribution.

File: akka-stream/src/main/scala/akka/stream/Graph.scala [hunk 1]
```diff
@@ -78,6 +78,12 @@ trait Graph[+S <: Shape, +M] {
    * less specific than attributes set directly on the individual graphs of the composite.
    */
   def addAttributes(attr: Attributes): Graph[S, M] = withAttributes(traversalBuilder.attributes and attr)
+
+  /**
+   * Get all attributes from this `Graph` which have added previously.
+   */
```
Comment (johanandren) [inline akka-stream/src/main/scala/akka/stream/Graph.scala:84]: I think the method name says all that needs to be said already and that this Scaladoc does not add anything, skip for the other overloads as well
```suggestion
```

**Review by johanandren — COMMENTED** [review]
Let's skip that superflous docsnippet and I think this is good to go

File: akka-stream/src/main/scala/akka/stream/Graph.scala (hunk 1 above)
Comment (jxnu-liguobin) [reply akka-stream/src/main/scala/akka/stream/Graph.scala:84]: ok, I will remove it

**Review by johanandren — APPROVED** [review]
LGTM, thanks!

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**